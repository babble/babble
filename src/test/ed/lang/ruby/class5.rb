
class Foo
  @@a = 1;

  def initialize( foo )
    puts foo
  end
    
  
  def blah
    return @@a;
  end
end

a = Foo.new(21);
puts a.blah

b = Foo.new(22);
puts b.blah
