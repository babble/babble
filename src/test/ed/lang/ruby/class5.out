21
1
22
1
